spec/models/user/user_relationship_extension_spec.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+RSpec.describe User::UserRelationshipExtension, type: :module do
+  let!(:user) { create(:user) }
+  let!(:followee1) { create(:user, email: 'followee1@example.com') }
+  let!(:followee2) { create(:user, email: 'followee2@example.com') }
+  let!(:follower1) { create(:user, email: 'follower1@example.com') }
+  let!(:follower2) { create(:user, email: 'follower2@example.com') }
+
+  let!(:active_relationship1) { ActiveRelationship.create(follower: user, followee: followee1) }
+  let!(:active_relationship2) { ActiveRelationship.create(follower: user, followee: followee2) }
```

Comment:
以下のようにすると、letが減らせると思った！
（letは極力減らすべきという考えが慣習的なので）

```suggestion
  let!(:active_relationships) do
    {
      1 => ActiveRelationship.create(follower: user, followee: followee1),
      2 => ActiveRelationship.create(follower: user, followee: followee2)
    }
  end
```